device.yml

```
@@ -0,0 +1,273 @@
+%YAML 1.1
+---
+# yaml-language-server: $schema=https://harp-tech.org/draft-02/schema/device.json
+device: MultiPwm
+whoAmI: 1040
+firmwareVersion: "1.1"
+hardwareTargets: "1.1"
+registers:
+  PwmChannel0Frequency: &pwmfreq
+    address: 32
+    type: Float
+    access: Write
+    description: Frequency (Hz) of PWM pulses in channel 0.
+  PwmChannel1Frequency:
+    <<: *pwmfreq
+    address: 33
+    description: Frequency (Hz) of PWM pulses in channel 1.
+  PwmChannel2Frequency:
+    <<: *pwmfreq
+    address: 34
+    description: Frequency (Hz) of PWM pulses in channel 2.
+  PwmChannel3Frequency:
+    <<: *pwmfreq
+    address: 35
+    description: Frequency (Hz) of PWM pulses in channel 3.
+  PwmChannel0DutyCycle: &pwmduty
+    address: 36
+    minValue: 0.1
+    maxValue: 99.9
+    defaultValue: 50
+    type: Float
+    access: Write
+    description: Duty cycle (0-100) of PWM pulses in channel 0.
+  PwmChannel1DutyCycle:
+    <<: *pwmduty
+    address: 37
+    description: Duty cycle (0-100) of PWM pulses in channel 1.
+  PwmChannel2DutyCycle:
+    <<: *pwmduty
+    address: 38
+    description: Duty cycle (0-100) of PWM pulses in channel 2.
+  PwmChannel3DutyCycle:
+    <<: *pwmduty
+    address: 39
+    description: Duty cycle (0-100) of PWM pulses in channel 3.
+  PwmChannel0NumPulses: &pwmnum
```

```suggestion
  PwmChannel0PulseCount: &pwmnum
```

The use of "number" is ambiguous (see [here](https://stackoverflow.com/a/6359175) but there are other references).